manifests/overlays/apps/base/elastic/elastic-app.yaml

```diff
@@ -0,0 +1,67 @@
+    <path d="M358.24 75.24C349.62 75.24 345.31 78.24 345.31 84.24C345.31 87.0134 346.31 88.97 348.31 90.11C350.31 91.25 354.814 92.4367 361.82 93.67C368.86 94.9034 373.84 96.6234 376.76 98.83C379.68 101.037 381.144 105.183 381.15 111.27C381.15 117.363 379.194 121.83 375.28 124.67C371.367 127.51 365.657 128.933 358.15 128.94C353.25 128.94 336.89 127.12 336.89 127.12L337.37 119.42C346.77 120.32 353.56 120.99 358.15 120.99C362.74 120.99 366.23 120.26 368.64 118.8C371.05 117.34 372.25 114.89 372.25 111.45C372.25 108.01 371.25 105.68 369.17 104.45C367.09 103.22 362.61 102.06 355.66 100.95C348.71 99.84 343.76 98.22 340.84 96.09C337.92 93.96 336.45 90 336.45 84.23C336.45 78.46 338.45 74.23 342.56 71.43C346.67 68.63 351.71 67.28 357.79 67.28C362.61 67.28 379.34 68.51 379.34 68.51V76.26C370.51 75.87 363.29 75.24 358.24 75.24Z" fill="#1C1E23"/>
+    <path d="M424 76.1801H405.15V104.52C405.15 111.313 405.643 115.777 406.63 117.91C407.63 120.05 409.97 121.11 413.68 121.11L424.23 120.4L424.83 127.75C420.835 128.489 416.79 128.926 412.73 129.06C406.57 129.06 402.303 127.557 399.93 124.55C397.557 121.543 396.373 115.813 396.38 107.36V76.1801H388V68.4801H396.42V50.3401H405.19V68.4801H424V76.1801Z" fill="#1C1E23"/>
+    <path d="M436.36 60.9101V50.5901H445.25V60.9101H436.36ZM436.36 128.48V68.4801H445.25V128.48H436.36Z" fill="#1C1E23"/>
+    <path d="M483.76 67.29C488.2 67.4795 492.622 67.9572 497 68.72L499.84 69.07L499.49 76.3C494.844 75.7114 490.172 75.3576 485.49 75.24C478.77 75.24 474.207 76.84 471.8 80.04C469.393 83.24 468.187 89.1667 468.18 97.82C468.18 106.487 469.307 112.513 471.56 115.9C473.813 119.287 478.513 120.987 485.66 121L499.66 119.94L500.02 127.29C494.57 128.213 489.065 128.768 483.54 128.95C474.3 128.95 467.92 126.577 464.4 121.83C460.88 117.083 459.12 109.083 459.12 97.83C459.12 86.57 461.017 78.6667 464.81 74.12C468.603 69.5734 474.92 67.2967 483.76 67.29Z" fill="#1C1E23"/>
+    </g>
+    <defs>
+    <clipPath id="clip0_15_21">
+    <rect width="500" height="171.38" fill="white"/>
+    </clipPath>
+    </defs>
+    </svg>
+  docsLink: https://www.elastic.co/platform
+  category: Self-managed
+  support: third party support
+  quickStart: ''
+  getStartedLink: https://www.elastic.co/guide/en/esre/current/index.html
+  csvName: elasticsearch-eck-operator-certified
```

Comment:
@dgutride Just a question here, the logic we have in place is using `startsWith`, just in case we have multiple csv that have the same prefix. I don't know wether we wanna add support from the Openshift ECK operator `elasticsearch-operator` too, in that case we might wanna change the `csvName`

Reply:
The name is on this line in their repo:  https://github.com/elastic/cloud-on-k8s/blob/8270ad0ae9aebbc39a31fede2c2eb90def222d5b/hack/operatorhub/config.yaml#L42